src/form/FormInput.js

```diff
@@ -149,13 +146,12 @@ const styles = StyleSheet.create({
     ...Platform.select({
       android: {
         minHeight: 46,
-        width: width - 30,
       },
       ios: {
         minHeight: 36,
-        width: width,
       },
     }),
+    width: '100%',
```

Comment:
@iRoachie This will only work on RN 0.43 or 0.42+ It will break for anyone not on that version. Is that something that was decided?